lib/groonga-query-log/command/run-regression-test.rb

```diff
@@ -420,10 +430,11 @@ def new_groonga_server
                           server_options)
       end
 
-      def format_report(success, elapsed_time)
+      def format_report(success, elapsed_time, n_execute_commands)
         formatted = format_elapsed_time(elapsed_time)
         if success
-          formatted << "Success"
+          formatted << "Number of execution commands: #{n_execute_commands}\n"
```

Comment:
Why do you show this only on success case?

Reply:
There is no special reason.
Therefore, I also displayed it in the failure case.